public/js/charts/trendsCoverage.js

```diff
@@ -0,0 +1,173 @@
+// NOTE: requires('sheets');
+// NOTE: requires('aq');
+
+window.aq.charts.trendsCoverage = {};
+
+window.Chart.defaults.global.defaultFontSize = 10;
+
+const acquisition = document.getElementById('acquisition');
+
+async function handleLocationChange() {
+  const nodes = await window.aq.getNodes();
+  const countyCitiesMap = await window.sheets.getCountyCitiesMap();
+  function getAvg(sensor) {
+    const total = sensor.reduce((acc, c) => acc + c, 0);
+    return total / sensor.length;
+  }
+  // Format dates from nodes
+  function formatDate(date, langCode) {
+    const dateObj = new Date(date);
+    const day = dateObj.toLocaleString(langCode, { day: '2-digit' }); // DD
+    const month = dateObj.toLocaleString(langCode, { month: 'short' }); // MMM
+    return `${day} ${month}`;
```

Comment:
Isn't this just
```suggestion
    dateObj.toLocaleString('en-GB', { day: '2-digit', month: 'short' })
```

Reply:
@kilemensi Yeah, I thought so too but it was returning the month first, then the date...something like `June 20` instead of `20 June` which is what we need for the chart

Reply:
**en-US** would return month first but _en-GB_ should return date first @nerdlet

Reply:
Ahhh @kilemensi Making the change